src/Dev/Install/InstallRequirements.php

```diff
@@ -279,9 +279,9 @@ public function check()
             null
         ));
 
-        $this->requireWriteable('mysite/_config/config.yml', array(
+        $this->requireWriteable('mysite/_config/theme.yml', array(
```

Comment:
Do we not need to write to `config.yml` anymore?

Reply:
I don't think so. There is no config.yml anymore.

Reply:
The config.yml's "project" is in mysite.yml and the "theme" config went to theme.yml. The file config.yml does not exist in 4.0 anymore.

Reply:
Correct, there's only `mysite.yml` and `theme.yml`, of which only `theme.yml` needs modification.